tests/pagination/build.gradle.kts

```diff
@@ -72,43 +71,36 @@ apollo {
   service("pagination.offsetBasedWithArray") {
     packageName.set("pagination.offsetBasedWithArray")
     srcDir("src/commonMain/graphql/pagination/offsetBasedWithArray")
-    @OptIn(ApolloExperimental::class)
     generateDataBuilders.set(true)
```

Comment:
Wait, did this become stable?

Reply:
No it's just that the Kotlin metadata is stripped and so the `OptIn` actually produces a warning

Reply:
Interesting... The `@Metadata` on the `@ApolloExperimental` annotation you mean? I always thought the annotation was the only thing checked by the compiler. Maybe `kotlin.RequiresOptIn` needs to be kept?

Reply:
Oh yeah not the Kotlin metadata - yes it's probably the `kotlin.RequiresOptIn` that should be kept, let me try

Reply:
Yes that works!
<img width="690" alt="Screenshot 2024-10-02 at 18 56 05" src="https://github.com/user-attachments/assets/bf8a893a-c340-4247-af50-70b13ea18a9f">